src/fiskaltrust.Launcher/Commands/ConfigCommand.cs

```diff
@@ -0,0 +1,248 @@
+
+            LauncherConfiguration? localConfiguration = null;
+            if (LauncherConfigurationFile is not null)
+            {
+                try
+                {
+                    localConfiguration = LauncherConfiguration.Deserialize(await File.ReadAllTextAsync(LauncherConfigurationFile));
+                    try
+                    {
+                        localConfiguration.Decrypt(CipyerAccessToken);
+                    }
+                    catch (Exception e)
+                    {
+                        Log.Warning(e, "Error decrypring launcher configuration file.");
+                    }
+                    localConfiguration.DisableDefaults();
+                }
+                catch (Exception e)
+                {
+                    Log.Error(e, "Could not read launcher configuration.");
+                }
+
+                if (localConfiguration is not null)
+                {
+                    Log.Information("Local configuration {LauncherConfigurationFile}\n{localConfiguration}", LauncherConfigurationFile, localConfiguration.Serialize(true, true));
+                }
+            }
+
+            if (LegacyConfigFile is not null)
+            {
+                LauncherConfiguration? legacyConfiguration = null;
+                try
+                {
+                    legacyConfiguration = await LegacyConfigFileReader.ReadLegacyConfigFile(LegacyConfigFile);
+                    legacyConfiguration?.Decrypt(CipyerAccessToken);
+                    legacyConfiguration?.DisableDefaults();
+                }
+                catch (Exception e)
+                {
+                    Log.Error(e, "Could not read legacy launcher configuration.");
+                }
+
+                if (legacyConfiguration is not null)
+                {
+                    Log.Information("Legacy configuration {LegacyConfigFile}\n{legacyConfiguration}", LegacyConfigFile, legacyConfiguration.Serialize(true, true));
+                }
+            }
+
+            CashBoxConfigurationFile ??= localConfiguration?.CashboxConfigurationFile;
+            if (CashBoxConfigurationFile is not null)
+            {
+                LauncherConfiguration? remoteConfiguration = null;
+                try
+                {
+                    remoteConfiguration = LauncherConfigurationInCashBoxConfiguration.Deserialize(await File.ReadAllTextAsync(CashBoxConfigurationFile));
+                    try
+                    {
+                        remoteConfiguration?.Decrypt(CipyerAccessToken);
+                    }
+                    catch (Exception e)
+                    {
+                        Log.Warning(e, "Error decrypring launcher configuration file.");
+                    }
+                    remoteConfiguration?.DisableDefaults();
+                }
+                catch (Exception e)
+                {
+                    Log.Error(e, "Could not read remote launcher configuration.");
```

Comment:
I think we're using this logic (decrypting the config) 3x in this file, might make sense to put it into a method